dbtmetabase/metabase.py

```diff
@@ -404,14 +434,192 @@ def build_metadata_lookups(
+                enumer = 1
+                while name in duplicate_name_check:
+                    name = f"{name}_{enumer}"
+                    enumer += 1
+                duplicate_name_check.append(name)
+
+                # Construct Exposure
+                description = (
+                    header
+                    + (
+                        "{}\n\n".format(description.strip())
+                        if description
+                        else "No description provided in Metabase\n\n"
+                    )
+                    + (
+                        "#### Query\n\n```\n{}\n```\n\n".format(
+                            "\n".join(
+                                line
+                                for line in self.native_query.strip().split("\n")
+                                if line.strip() != ""
+                            )
+                        )
+                        if self.native_query
+                        else ""
+                    )
+                    + "#### Metadata\n\n"
+                    + ("Metabase Id: __{}__\n\n".format(item["id"]))
+                    + ("Created On: __{}__".format(created_at))
+                )
+                captured_exposure = {
+                    "name": name,
+                    "description": description,
+                    "type": "analysis" if item["model"] == "card" else "dashboard",
+                    "url": f"{self.protocol}://{self.host}/{item['model']}/{item['id']}",
+                    "maturity": "medium",
+                    "owner": {
+                        "name": creator_name,
+                        "email": creator_email,
+                    },
+                    "depends_on": [
+                        refable_models[exposure.upper()]
+                        for exposure in list({m for m in self.models_exposed})
+                        if exposure.upper() in refable_models
+                    ],
+                }
+                captured_exposures.append(captured_exposure)
+
+        # Output dbt YAML
+        with open(
+            os.path.expanduser(os.path.join(output_path, f"{output_name}.yml")), "w"
+        ) as docs:
+            yaml.dump(
+                {"version": 2, "exposures": captured_exposures},
+                docs,
+                allow_unicode=True,
+                sort_keys=False,
+            )
+        return {"version": 2, "exposures": captured_exposures}
+
+    def _extract_card_exposures(self, card_id: int, model: Optional[Mapping] = None):
+        if not model:
+            model = self.api("get", f"/api/card/{card_id}")
+        query = model.get("dataset_query", {})
+        if query.get("type") == "query":
+            source_table_id = query.get("query", {}).get(
+                "source-table", model.get("table_id")
+            )
+            if source_table_id in self.table_map:
+                if isinstance(source_table_id, str) and source_table_id.startswith(
```

Comment:
Funny formatting, maybe move `startswith` to the new line altogether.

Reply:
simplified conditional to just use startswith on str(source_table_id) instead of nesting conditional check for str